### PR TITLE
Bitcoin Death Cross demo

### PR DESCRIPTION
**Pull request by neilstevenson**
Please SQUASH Merge

I need it pushed to main repo so I can correctly set image URLs in README.md

### CODE REVIEW

File: bitcoin-death-cross/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# bitcoin-death-cross
+
+This demo only works on a single member
```
Comment (eminn) [inline bitcoin-death-cross/README.md]: Can we enhance this README to include build and run instructions. Also high level overview would be also helpful

File: bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	 * <p>Run one copy of the Moving Average job
+	 * in this cluster.
+	 * </p>
+	 */
+	@Override
+	public void run(String... args) throws Exception {
+		String prefix = this.getClass().getSimpleName() + " -";
+		
+		Pipeline pipeline = MovingAverage.build();
+    	JobConfig jobConfig = new JobConfig();
```
Comment (eminn) [inline bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java]: I don't whether this formatting errors come from the file or github couldn't render the file correctly ? Discard this if the file is properly formatted

File: bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+ * <p><b>Note:</b> {@code Task1}, {@code Task2} and {@code Task3}
+ * are ordered to run before {@code Task4}. The first three can
+ * be run in any order, all will appear to do nothing until the
+ * fourth starts producing data.
+ * </p>
+ */
+@Component
+@Order(MyConstants.PRIORITY_ONE)
+@Slf4j
+public class Task1 implements CommandLineRunner {
```
Comment (eminn) [inline bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java]: Can we change the Task1, Task2, Task3, Task4 names to more meaningful ones ?

Comment (neilstevenson) [reply bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java]: Done. Have renamed `Task1` to `Task1JetJob`, `Task2` to `Task2ChartPanel`, `Task3` to `Task3TopicListener` and `Task4` to `Task4PriceFeed`. I wanted to keep the numbering in there as there is a partial ordering to the tasks, but indicating what each does is a winner.

File: bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java (hunk 2 above)
Comment (neilstevenson) [reply bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java]: Fixed in e5ed844

File: bitcoin-death-cross/README.md (hunk 1 above)
Comment (neilstevenson) [reply bitcoin-death-cross/README.md]: There will be a blog.hazelcast.com entry, and once published I'll copy it to README.md

**Pull request merged**